projects/Magnetosphere/Magnetosphere.h

```diff
@@ -117,5 +118,7 @@ namespace projects {
    }; // class Magnetosphere
 } // namespace projects
 
+extern SysBoundary sysBoundaryWrapper;
```

Comment:
I don't quite see yet why this separate object is necessary. Is it just so that the existence of boundaries can be tested through a global object somewhere?

Renaming all the sysboundary accesses further down in vlasiator.cpp will probably again lead to conflicts with the time varying inflow branch (although they won't be hard to correct)

Reply:
Yep, I think passing sysboundaries down to all projects doesn't make much sense for this special case, so a global object was my preferred choice.
The naming may indeed cause some clash, but because it's been just a local object and not part of an interface, it should be minor (and I'd say helpful).